src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
+    private String color;
+    private double firstLeg;
+    private double secondLeg;
```

Comment:
The class RightTriangle should not begin with an empty line. Please remove the empty line at the start of the class definition.